saltgui/static/stylesheets/main.css

```diff
@@ -19,14 +19,17 @@ header {
   border-top: 2px solid #4caf50;
   background-color: white;
   margin-top: 0;
-  padding: 12px 24px 12px 24px;
+  height: 51px;
```

Comment:
That fixed height should not be needed. It is a bit arbitrary value. Also it causes a gap between the menu title item and the dropdown panel. It looks much better when I disable it.